### ISSUE TITLE
[Bug]: Can't unlink site

### ISSUE THREAD

**Issue by ericmp33** [beyondcode/herd-community#515]
### Operating system version

Windows 11

### System architecture

Intel (x86)

### Herd Version

1.0.0

### PHP Version

PHP 8.3.4 (cli) (built: Mar 13 2024 11:42:47) (NTS Visual C++ 2019 x64)

### Bug description

I created a site called `example-app`. I've linked it and seems i can't unlink it. It says that is not linked but is link it.

After trying it several times, I thought maybe doesn't work because of the `-` between `example` and `app`. So I tried it creating a site called `test`. But same behavior.

![image](https://github.com/beyondcode/herd-community/assets/60661635/ac9af9d3-95ca-4d1a-83a7-63c48bce80e0)

### Steps to reproduce

1. Create a site
2. Link it
3. Try to unlink it

**Comment by isimmons** [comment]
Maybe some helpful info and a [related discussion](https://github.com/beyondcode/herd-community/discussions/368)

I created a site in the UI, not in the CLI so I did not run the link step. Nothing ever showed up in `.config/herd/config/valet/Sites/` but everything worked. I'm guessing the UI does something different for this step. After creating the site in the UI, running unlink in the CLI shows the success message even though there never was a link in that valet/Sites directory.

**Comment by martin55** [comment]
I have probably related problem on Win11: 
CLI command: `_herd link [domain_name]_` doesn't do anything, but a success message is returned. (returned message: A symbolic link has been created in...). Linkage does not work, and nothing is created in `_.config/herd/config/valet/Sites_`

Did anybody manage to get working multiple domains for a site?

**Comment by mpociot** [comment]
Yeah, the problem is that `herd link` requires an elevated shell. I'll adjust the CLI so that it prompts for admin access and shows an error message if that's not the case.

**Comment by helloreece** [comment]
@mpociot I have tried with admin shell. The experience still happens... it confirms creating the link, but doesn't display it in the links table. The site is also not accessible, so I assume the link was not actually generated, even with the elevated shell.

![image](https://github.com/beyondcode/herd-community/assets/129170707/2e951c8d-af97-44cb-ba7a-f7f0b6a9dc80)

**Comment by mpociot** [comment]
Can you manually add the `%USERPROFILE%\.config\herd\config\valet\Sites` folder to the parked paths?
Can you also verify that the symlink exists in that folder? :)

**Comment by martin55** [comment]
> Can you manually add the `%USERPROFILE%\.config\herd\config\valet\Sites` folder to the parked paths? Can you also verify that the symlink exists in that folder? :)
When I run `herd link [domain]` it actually adds `%USERPROFILE%/.config/herd/config/valet/Sites` path automatically. It's visible also at Herd UI. Added path contains slashes NOT backslashes. I even tried to change those to backslashes, but I think bigger issue is that no symlinks are created in the folder.

**Comment by mpociot** [comment]
Gotcha - thanks. 
You can create a symlink manually as a workaround. Will be fixed in the next version

**Comment by martin55** [comment]
You need to open command prompt (not PowerShell) and then create symlink. Also be sure herd path is correct (use backslashes)

`mklink \d [symlink] [target]`

> Creates a symbolic link.
> 
> MKLINK [[/D] | [/H] | [/J]] Link Target
> 
>         /D      Creates a directory symbolic link.  Default is a file
>                 symbolic link.
>         /H      Creates a hard link instead of a symbolic link.
>         /J      Creates a Directory Junction.
>         Link    Specifies the new symbolic link name.
>         Target  Specifies the path (relative or absolute) that the new link
>                 refers to.

**Comment by helloreece** [comment]
```
C:\Windows\System32>mklink /d C:\Users\reece\Herd\speakup.manage-my-course C:\Users\reece\Herd\manage-my-course
symbolic link created for C:\Users\reece\Herd\speakup.manage-my-course <<===>> C:\Users\reece\Herd\manage-my-course
```

![image](https://github.com/beyondcode/herd-community/assets/129170707/8605c0d7-1048-4841-a524-dcc42a3c424e)

![image](https://github.com/beyondcode/herd-community/assets/129170707/590bdec6-1ecb-4074-8daa-9b2eef06095d)

**Comment by mpociot** [comment]
You'll need to manually add the entry to your hosts file for the symlink in this case.

**Comment by helloreece** [comment]
Worked :) thanks!

**Comment by mpociot** [comment]
Fixed in version 1.0.1 
https://herd.laravel.com/changelog/windows#101
